python/python/glide/constants.py

```diff
@@ -25,6 +25,9 @@
     bytes,
     Dict[bytes, "TResult"],
     Mapping[bytes, "TResult"],
+    bytes,
```

Comment:
You have bytes twice. And Mapping. And mess between `"TResult"` and `T`.
I propose:
1. Replace `str` and `bytes` both by `TEncodable`
2. Replace `Mapping[str, "TResult"]` and `Mapping[bytes, "TResult"]` by `Mapping[TEncodable, "TResult"]`